### ISSUE TITLE
expansion panel scroll infinitely down

### ISSUE THREAD

**Issue by ranbuch** [angular/material2#9566]
#### Bug, feature request, or proposal:
I suspect that if the user is scrolled all the way down there's some infinite loop that endlessly scrolls down.

#### What is the expected behavior?
I would expect that the user would be able to scroll up is he chooses to.

#### What is the current behavior?
The user cannot scroll up if he reaches to the end of the page (if he scrolls all the way down).

#### What are the steps to reproduce?
To reproduce you can go to this site:
https://staging.creators3d.com/
and use those credentials:
ranbuch@hotmail.com
r123456
Than you can see the issue on this page:
https://staging.creators3d.com/job/665
Just scroll all the way down and than try to scroll up again.

#### Which versions of Angular, Material, OS, TypeScript, browsers are affected?
@angular/common@5.1.1
@angular/material@5.0.1

**Comment by josephperrott** [comment]
It looks like the credentials you provided to view the reproduction does not work.  

Can you provide a reproduction in either [Plunker](https://goo.gl/uDmqyY) or [StackBlitz]( https://goo.gl/wwnhMV)

**Comment by ranbuch** [comment]
My bad,
It's:
ranbuch@gmail.com
r123456

**Comment by josephperrott** [comment]
I definitely see the issue you are talking about in this page specifically, however there is no way to determine if the issue is coming from a specific component.  Many Material components (and I assume application components specific as well) are used in the page, so its hard to know what components are at play.

Can you provide a reproduction in either [Plunker](https://goo.gl/uDmqyY) or [StackBlitz]( https://goo.gl/wwnhMV).  This will allow us to isolate the issue as well as see the code which causing the issue.

**Comment by ranbuch** [comment]
Couldn't reproduce in Plunker (works fine there) but I've published a new version of my Application without the `--prod` flag.
I've clicked to the `body` element on the `devtools` and under `Event Listeners` => `scroll` there's only 1 event.
This event listener is coming from `zone.js` file.
There's indeed an infinite loop there.
I'm able to scroll the page up using touch though.

I do belie it's related to `Material` and most probably to `expansion panel` but I could be wrong.
Unfortunately I have no idea on how to isolate this issue further.
